### PR TITLE
Extend the documentation for oomd a bit

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/systemd-oomd.service.xml [hunk 1]
```diff
@@ -29,47 +29,62 @@
...
+    <para>You can enable monitoring and actions on units by setting <varname>ManagedOOMSwap=</varname> and
+    <varname>ManagedOOMMemoryPressure=</varname> in the unit configuration, see
+    <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
+    <command>systemd-oomd</command> retrieves information about such units from <command>systemd</command>
+    when it starts and watches for subsequent changes.</para>
+
+    <para>Cgroups of units with <varname>ManagedOOMSwap=</varname> or
+    <varname>ManagedOOMMemoryPressure=</varname> set to <option>kill</option> will be monitored.
+    <command>systemd-oomd</command> periodically polls PSI statistics for the system and those cgroups to
+    decide when to take action. If configured limits are exceeded, <command>systemd-oomd</command> will
```
Comment (daandemeyer) [inline man/systemd-oomd.service.xml]: "If the configured limits ..."

File: man/systemd-oomd.service.xml [hunk 2]
```diff
@@ -29,47 +29,62 @@
...
+    select a cgroup to terminate, and send <constant>SIGKILL</constant> to all processes in it. Note that
+    only descendant cgroups are eligible candidates for killing; the unit with its property to
```
Comment (daandemeyer) [inline man/systemd-oomd.service.xml]: "with its property set to ..."

File: man/systemd.service.xml [hunk 3]
```diff
@@ -1123,8 +1123,14 @@
       <varlistentry>
         <term><varname>OOMPolicy=</varname></term>
 
-        <listitem><para>Configure the Out-Of-Memory (OOM) killer policy. On Linux, when memory becomes scarce
-        the kernel might decide to kill a running process in order to free up memory and reduce memory
+        <listitem><para>Configure the out-of-memory (OOM) kernel killer policy. Note that the userspace OOM
+        killer
+        <citerefentry><refentrytitle>systemd-oomd.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        is a more flexible solution that is allows preventing out-of-memory situations for the userspace, not
```
Comment (bluca) [inline man/systemd.service.xml]: /that is allows/that allows/

**Review by bluca — APPROVED** [review]

**Comment by anitazha** [comment]
LGTM besides the grammar issue bluca pointed out. Thank you!

File: man/systemd.service.xml (hunk 3 above)
Comment (keszybz) [reply man/systemd.service.xml]: I reworded that as "that aims to prevent". I don't want to say "prevents", because it can't cover all cases.

**Comment by keszybz** [comment]
Updated.

**Comment by poettering** [comment]
lgtm too

**Pull request merged**